alicja/config/proto/interface.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
```

Comment:
Add a comment at the top of the file describing what it does/what its purpose is.